### PR TITLE
OCPBUGS#14864_4_11:Add known issue Selinux relabeling issue

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.11 only

Issue: https://issues.redhat.com/browse/OCPBUGS-14864

Link to docs preview: https://file.rdu.redhat.com/lpettyjo/OCPBUGS-14864_4_11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-known-issues

QE review:
- [x] QE has approved this change.

Additional information:

**PTAL**: @gcharot, @chao007, @sferich888, @xltian, @mwerner2113, @gnufied

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -2509,6 +2509,8 @@ openshift.io/scc: hostmount-anyuid
 
 * Currently, a certificate compliance issue, specifically outputted as `x509: certificate is not standards compliant`, exists when you run the installation program on macOS for the purposes of installing an {product-title} cluster on VMware vSphere. This issue relates to a known issue with the `golang` compiler in that the compiler does not recognize newly supported macOS certificate standards. No workaround exists for this issue. (link:https://issues.redhat.com/browse/OSDOCS-5694[*OSDOCS-5694*])
 
+* Currently, when using a persistent volume (PV) that contains a very high file count, the pod may not start or takes an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-11-release-notes.adoc:2512]:
```suggestion
* Currently, when using a persistent volume (PV) that contains a very high file count, the pod may not start or may take an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
```

**Review by jeana-redhat — COMMENTED** [review]
Same as 4.12

**Comment by gnufied** [comment]
Other than minor language comment on other PR. lgtm

**Comment by gcharot** [comment]
lgtm

**Comment by xltian** [comment]
lgtm

**Comment by kalexand-rh** [comment]
LGTM

**Pull request merged**